env/dev/clj/user.clj

```diff
@@ -1,6 +1,9 @@
 (ns user
   (:require [mount.core :as mount]
-            [commiteth.figwheel :refer [start-fw stop-fw cljs]]))
+            [commiteth.figwheel :refer [start-fw stop-fw cljs]]
+            [clojure.tools.namespace.repl :as repl :refer [refresh]]))
+
+(repl/set-refresh-dirs "src" "dev" "test")
```

Comment:
Why exactly is this needed? Isn't the classpath correct?

Reply:
In some cases `clojure.tools.namespace` will try to reload code from libraries and break. There are also some issues when reloading cljc files. This prevents that from happening.

Reply:
Could you please explain those two cases (or point to some page that explains them) and how that `set-refresh-dirs` with `"src" "dev" "test"` solves them? Would like to understand it well.

Reply:
You can see the issue if you run `clojure.tools.namespace.repl/refresh-all` on the repl a couple of times. More info [here](https://github.com/clojure/tools.namespace#reloading-code-usage) and [here](https://github.com/stuartsierra/component/issues/45#issuecomment-260974812).

Reply:
Okay with it, understood it more, thanks for the links.